IBAnimatable/UIBezierPathExtension.swift

```diff
@@ -0,0 +1,171 @@
+//
+//  Created by phimage on 18/05/2017.
+//  Copyright © 2017 IBAnimatable. All rights reserved.
+//
+
+import UIKit
+
+extension UIBezierPath {
+
+  // MARK: - Circle
+
+  /**
+   Create a Bezier path for a circle shape.
+   
+   - Parameter bounds: The bounds of shape.
+   */
+  convenience init(circleIn bounds: CGRect) {
```

Comment:
I see that the XXXExtension.swift files contain public extension
but here is this no need to share externally this code